hostscripts/upgrade-scale/README.md

```diff
@@ -75,243 +75,41 @@ rm -r automation.old
 mv automation automation.old
 git clone --branch=upgrade-scale https://github.com/SUSE-Cloud/automation
 # use that before the PR is merged
-#git clone --branch=upgrade-scale-scenario https://github.com/skazi0/automation
+#git clone --branch=upgrade-scale-script https://github.com/skazi0/automation
```

Comment:
Does this need to be part of the script? I think it's enough in PR description, if needed

Reply:
yeah, last time we merged the PR only after it was not used anymore so I kept this line for easier access... it should have been removed before merging.